crypto/fipsmodule/cpucap/cpucap.c

```diff
@@ -82,3 +82,6 @@ HIDDEN uint32_t OPENSSL_armcap_P = 0;
 HIDDEN uint8_t BORINGSSL_function_hit[7] = {0};
 #endif
 
+// This variable is used only for testing purposes to ensure that the library
+// constructor is executed and the capability variable is initialized.
+HIDDEN uint8_t OPENSSL_cpucap_initialized = 0;
```

Comment:
Not sure I like all these variables. But I can't come up with a better way to do it myself... so...